Remove reference to Universal Analytics from README (#371)
GoogleChrome/web-vitals: README.md

### Send the results to Google Analytics

> Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique dimension value (in this case, the `metric_id`, as shown in the examples below) on every metric instance that you send to Google Analytics, you can create a report yourself using the [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting) and any data visualization library you choose.

Reviewer: The Reporting API is for UA only. The Data API is the API for GA4, which is still in alpha/beta: https://developers.google.com/analytics/devguides/reporting/data/v1

Also it's  not clear to me whether "unique dimension value" is the right terminology given that this is going to be GA4 only; it might be "unique event parameter. However, I'm not 100% sure how you get those values via the API. It might still require creating a custom dimension in the UI, which is then defined as a custom event parameter value (again, not 100% sure).

Alternatively, we could just not recommend the API at all and stick to the recommendation of using BigQuery...

Author: OK let's just remove this section.

For details of have to query this data in [BigQuery](https://cloud.google.com/bigquery), or visualise it in [Looker Studio](https://lookerstudio.google.com/), see the [Measure and debug performance with Google Analytics 4 and BigQuery](https://web.dev/vitals-ga4/) post.

> #### Using `gtag.js` (Google Analytics 4)

Reviewer: This heading can probably be removed as well, since there's now only going to be one option.

Author: Done.

### Send the results to Google Analytics

> Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique dimension value (in this case, the metric `id`, as shown in the examples below) on every metric instance that you send to Google Analytics, you can create a report yourself using the [Google Analytics Reporting API](https://developers.google.com/analytics/devguides/reporting) and any data visualization library you choose.

Reviewer: Overall I think that intro content is still important to keep, especially the bit about GA not supporting distributions (and thus no p75). I just want to make sure we're using the correct terminology. I'm 99% sure the Data API can do this, what I'm not clear on is whether or not the API can query the event param directly or if you need to register a custom dimension in the UI first.

Note: I'm not suggesting we figure this all out in this PR, but I also want to make sure we're not saying anything incorrect. What about something like:

> Google Analytics does not support reporting metric distributions in any of its built-in reports; however, if you set a unique event parameter value (in this case, the `metric_id`, as shown in the example below) on every metric instance that you send to Google Analytics, you can create a report yourself by first getting the data via the [Google Analytics Data API](https://developers.google.com/analytics/devguides/reporting/data/v1) or via [BigQuery export](https://support.google.com/analytics/answer/9358801) and then visualizing it any charting library you choose.

Author: LGTM!

> For details of have to query this data in [BigQuery](https://cloud.google.com/bigquery), or visualise it in [Looker Studio](https://lookerstudio.google.com/), see the [Measure and debug performance with Google Analytics 4 and BigQuery](https://web.dev/vitals-ga4/) post.

Reviewer: ```suggestion
For details on how to query this data in [BigQuery](https://cloud.google.com/bigquery), or visualize it in [Looker Studio](https://lookerstudio.google.com/), see [Measure and debug performance with Google Analytics 4 and BigQuery](https://web.dev/vitals-ga4/).
```

Also, I'd put this content after the code example.

Author: Done.
